napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -161,6 +165,9 @@ def set_curr_img(self):
         curr_img : Dict[str, str]
             The current image {'File Path' : 'path', 'Row' : str(rownum)}
         """
+        if self._annotation_model.get_annotations() is None:
+            return
+
```

Comment:
`if not self._annotation_model.get_annotations() is None`

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/60/commits/2b55506cb86f9b44cd3aa371723c19f18d7576c9